### ISSUE TITLE
gmaps breaks click events

### ISSUE THREAD

**Issue by reneolivo** [assaf/zombie#913]
This probably has nothing to do with Zombie itself, but I don't seem to find any documentation or work around of this issue anywhere.

My website loads the gmaps script from this [repo](https://github.com/hpneo/gmaps/tree/0.2.30) and when I use `browser.click` I get a `[RangeError: Maximum call stack size exceeded]` error message. Usually I can bypass this issue by doing this:

```
browser.click('element', function(err) {
   //ignore errors
   continue();
});
```

But the `browser.on('alert', ...);` doesn't seem to catch any events when this error message is reproduced. The click event has nothing to do with the map and as soon as I remove the gmaps library and all dependencies the alert events work again.

Any ideas why gmap is polluting the click event? Is there a way to remove those files on the fly? 

Thanks.

**Comment by assaf** [comment]
I think you mean `on('error', fn)`, and no, there's no possible way to catch this error. This error is caused by infinite recursion, since the stack has been exhausted, Node can't safely unwind it and return control to the error handler. It should be reported as `process.on('uncaughtException`, fn)`

**Comment by reneolivo** [comment]
No, I don't mean `on('error', fn)`. This is what I want to accomplish:

1. browser opens the page.
2. browser fills an input.
3. browser clicks on a submit button.
4. browser waits for an alert message.

The page has a Google Map widget which I'm not interested in testing. When the widget is present, all steps run, but no alert message is carried out. When I do this:

```
browser.click('#submit-button', function(err) {
  if (err) throw err;
});
```
I get the `[RangeError: Maximum call stack size exceeded]` and it seems that the submit action is not carried out. When I add a `$('#submit-button').click()` at the end of my page for testing purposes, the `browser.on('alert', fn)` works fine.

After digging around I noticed the error only happens when the Google Map script is loaded. When I use Chrome or Firefox I can't reproduce this error. Since I don't need to test any Google Maps related functionality I have opted for removing the the Google Map script when doing the automated test.

I was adding the issue in case anyone has had any similar experience (since there's nothing online) and if anyone had a workaround. For now I'm happy with my solution, but I can see it being a problem if my page relies heavily on GMaps.
